chatmd/local_sync.py

```
@@ -0,0 +1,197 @@
+            output_dir=output_dir,
+            collapse_threshold=collapse_threshold,
+            html=html,
+            html_theme=html_theme,
+            **importer_kwargs,
+        )
+        if diff and snapshot is not None:
+            result.diff_path = write_delta_diff(snapshot, result.markdown_path)
+            try:
+                snapshot.unlink()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)